addons/web/static/src/js/views/graph/graph_view.js

```diff
@@ -141,6 +142,15 @@ var GraphView = AbstractView.extend({
         this.loadParams.comparisonDomain = params.comparisonDomain;
         this.loadParams.stacked = this.arch.attrs.stacked !== "False";
     },
+
+     /**
+     *
+     * @override
+     */
+    getRenderer(parent, props) {
+        props = Object.assign(props || {}, this.rendererParams);
+        return new RendererWrapper(null, this.config.Renderer, props);
+    },
 });
 
```

Comment:
missing props validation

Reply:
Pivot renderer has not. I think it would be too long

Reply:
grid renderer does

Reply:
will do